lib/taskana-core-test/src/test/java/acceptance/task/claim/SetOwnerAccTest.java

```diff
@@ -0,0 +1,419 @@
+
+    BulkOperationResults<String, TaskanaException> results =
+        taskService.setOwnerOfTasks("someUser", taskIds);
+
+    assertThat(results.containsErrors()).isFalse();
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_SetOwnerOfTasks_When_TasksAreDuplicated() throws Exception {
+    Task taskReadyForReview =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassificationSummary)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .state(TaskState.READY_FOR_REVIEW)
+            .buildAndStore(taskService, "admin");
+    Task taskReady =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassificationSummary)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .state(TaskState.READY)
+            .buildAndStore(taskService, "admin");
+    List<String> taskIds =
+        List.of(
+            taskReady.getId(),
+            taskReadyForReview.getId(),
+            taskReady.getId(),
+            taskReadyForReview.getId());
+
+    BulkOperationResults<String, TaskanaException> results =
+        taskService.setOwnerOfTasks("someUser", taskIds);
+
+    assertThat(results.containsErrors()).isFalse();
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_SetOwnerOfTasks_When_TasksAreDuplicatedAndTaskNotExisting() throws Exception {
+    Task taskReadyForReview =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassificationSummary)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .state(TaskState.READY_FOR_REVIEW)
+            .buildAndStore(taskService, "admin");
+    Task taskReady =
+        TaskBuilder.newTask()
+            .classificationSummary(defaultClassificationSummary)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .primaryObjRef(DefaultTestEntities.defaultTestObjectReference().build())
+            .state(TaskState.READY)
+            .buildAndStore(taskService, "admin");
+    List<String> taskIds =
+        List.of(
+            taskReady.getId(),
+            "TKI:000000000000000000000000000047110059",
+            taskReadyForReview.getId(),
+            taskReady.getId());
+
+    BulkOperationResults<String, TaskanaException> results =
+        taskService.setOwnerOfTasks("someUser", taskIds);
+
+    assertThat(results.containsErrors()).isTrue();
+    assertThat(results.getErrorMap()).hasSize(1);
+    assertThat(results.getErrorForId("TKI:000000000000000000000000000047110059"))
+        .isInstanceOf(TaskNotFoundException.class);
+  }
+
+  @WithAccessId(user = "user-b-2")
+  @Test
+  void should_ThrowExceptions_When_SetOwnerOfMultipleTasksAreNotAuthorizedOnWorkBasket()
```

Comment:
What do you think about a new name should_ReturnExceptions_When_SettingOwnerOfMultipleTasksWhileNotAuthorized() ?